lib/abci/handlers/beginBlockHandlerFactory.js

```diff
@@ -25,6 +27,8 @@ function beginBlockHandlerFactory(
    * @return {Promise<abci.ResponseBeginBlock>}
    */
   async function beginBlockHandler({ header }) {
+    logger.debug(`Starting processing of a block at height ${header.height}`);
```

Comment:
Let’s keep it short. It will ve easier to scan logs. Something like: ”Block begin #height”